doc/tutorials/source_tracking.rst

```diff
@@ -20,7 +20,7 @@ Test Usage
 ----------
 Before running SourceTracker check to see if it installed and accessible to QIIME::
 
-    print_qiime_config.py -t
+    print_qiime_config.py -tfv
```

Comment:
The specific message that the documentation suggests you look for is
`sourcetracker is installed ... ok` which only comes up if you add the
-v flag.

On (Jun-03-15| 6:04), Greg Caporaso wrote:
> > @@ -20,7 +20,7 @@ Test Usage
> >  ----------
> >  Before running SourceTracker check to see if it installed and accessible to QIIME::
> >  
> > -    print_qiime_config.py -t
> > +    print_qiime_config.py -tfv
> 
> What does the ``v`` do here? I think just ``-tf`` gets it. 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiime/pull/2039/files#r31619645